llvm/lib/Target/AIE/Utils/AIELoopUtils.cpp

```diff
@@ -32,6 +32,27 @@ std::optional<int64_t> getMinTripCount(const MachineBasicBlock &LoopBlock) {
   return getMinTripCount(getLoopID(LoopBlock));
 }
 
+std::optional<bool> getPipelinerDisabled(const MachineBasicBlock &LoopBlock) {
+  auto *LoopID = getLoopID(LoopBlock);
+  if (!LoopID) {
```

Comment:
I think it is better to keep the same brace convention for all single-sentence loops

Reply:
I agree. I will add braces everywhere.